knative-operator/hack/update-deps.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+readonly ROOT_DIR=$(dirname $0)/..
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# Prune modules.
+go mod tidy
+go mod vendor
+
+# TODO: Remove this once we bump kubernetes versions.
```

Comment:
So iiur, the patch is needed because the upstream manifestival only works with new k8s versions. But why are we need not pinning the manifestival also to an older version that works with k8s 1.13 instead of patching it on the fly ?

Reply:
I fighted with go modules quite a while to make this work with the older manifestival version, but it would throw errors at me. This seemed the most non-intrusive way forward for now.